AIDevGallery.SourceGenerator/ScenariosSourceGenerator.cs

```diff
@@ -94,6 +94,7 @@ internal static partial class ScenarioCategoryHelpers
                                         ScenarioType = ScenarioType.{{scenarioCategory.Key}}{{scenario.Key}},
                                         Name = "{{scenario.Value.Name}}",
                                         Description = "{{scenario.Value.Description}}",
+                                        Instructions = "{{scenario.Value.Instructions}}",
```

Comment:
The way this is setup, you would create an empty string, not a null string, but the type is defined as nullable in the Scenario class (in the SG project). Lets just make sure typing there is meaningful. It seems like this property shouldn't be null, so if it is we should fail, not write "". If, on the other hand, the type should be nullable (like how it is defined), the type in the main project should not be initialized as "", but rather with null.